frontend/templates/views/examples/manual.j2

```diff
@@ -1,56 +1,39 @@
 {% extends '/layouts/default.j2' %}
```

Comment:
Can we rename the template file to `documentation.j2`

Reply:
Fixed with 8e8b6e7.